festipals/src/components/Pals.js

```diff
@@ -1,10 +1,13 @@
 import React, { Component } from 'react';
+import samplePals from './../samplePals.json';
```

Comment:
I don't think we have to include those here, as the data can be passed to a Pal as a prop. (Read next comment)